### PR TITLE
Arabic/Kurdish updates

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: channels/tr.m3u [hunk 1]
```diff
@@ -15,6 +15,10 @@ https://live.al-zahratv.com/live/playlist2/index.m3u8
 http://stream.taksimbilisim.com:1935/altastv/bant1/playlist.m3u8
 #EXTINF:-1 tvg-id="ARASTV.tr" tvg-name="ARAS TV" tvg-country="TR" tvg-language="" tvg-logo="" group-title="",ARAS TV
 http://1.rtmp.org/tv217/yayin.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="TR" tvg-language="Arabic" tvg-logo="" group-title="",Belqees TV
+https://svs.itworkscdn.net/itwlive/itw3.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="TR" tvg-language="Arabic" tvg-logo="" group-title="",Belqees TV
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/c/BelqeesTV/live
```
Comment (freearhey) [inline channels/tr.m3u:21]: Belqees TV is a Yemeni channel according to this: https://www.lyngsat.com/tvchannels/ye/Belqees-TV.html

File: channels/qa.m3u [hunk 2]
```diff
@@ -1,4 +1,6 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="AlJazeera.qa" tvg-name="Al Jazeera" tvg-country="QA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al Rassoul
```
Comment (freearhey) [inline channels/qa.m3u:2]: Wrong tvg-id and tvg-name.

File: channels/ir.m3u [hunk 3]
```diff
@@ -107,6 +107,8 @@ https://live.kalemehtv.tv/live/ngrp:kalemeh_all/playlist.m3u8
 https://stream.alkafeel.net/live/alkafeel/rAa5PGot1/manifest.m3u8
 #EXTINF:-1 tvg-id="KarbalaLive4.ir" tvg-name="Karbala Live 4" tvg-country="IR" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/Karbala_tv.jpg" group-title="Religious",Karbala Live 4 (360p)
 https://stream.alkafeel.net/live/alkafeel/rAa5PGot4/manifest.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Karbala" tvg-country="IR" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/Karbala_tv.jpg" group-title="Religious",Karbala TV
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/c/karbalaatv/live
```
Comment (freearhey) [inline channels/ir.m3u:111]: Karbala Satellite Channel is Iraqi channel.

https://www.youtube.com/c/karbalaatv/about
https://www.lyngsat.com/tvchannels/iq/Karbala-Satellite-Channel.html

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/qa.m3u (hunk 2 above)
Comment (tilfizyawn) [reply channels/qa.m3u]: Thanks fixed

File: channels/tr.m3u (hunk 1 above)
Comment (tilfizyawn) [reply channels/tr.m3u]: Thanks fixed

File: channels/ir.m3u (hunk 3 above)
Comment (tilfizyawn) [reply channels/ir.m3u]: Thanks fixed

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]

**Review by freearhey — CHANGES_REQUESTED** [review]
https://github.com/iptv-org/iptv/pull/2967#discussion_r642577586

File: channels/eg.m3u [hunk 4]
```diff
@@ -17,6 +19,8 @@ https://5d12bc59c4748.streamlock.net/redirect/ctvchannel.tv/ctv.smil?type=m3u8
 http://media3.smc-host.com:1935/cycnow.com/smil:cyc.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="ElsharqTV.eg" tvg-name="Elsharq TV" tvg-country="EG" tvg-language="Arabic" tvg-logo="http://elsharq.tv/img/elsharq.png" group-title="General",Elsharq TV (1080p)
 https://mn-nl.mncdn.com/elsharq_live/live/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="EG" tvg-language="Arabic" tvg-logo="https://copticocc.org/mesat/wp-content/uploads/2018/07/logo-width12.png" group-title="Religious",MESat
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/channel/UCg5uHOxrP5GkMWldOavPKGQ/live
```
[the remark this answers is not in the document]
Comment (Dum4G) [reply channels/eg.m3u:23]: @LaneSh4d0w

[the remark this answers is not in the document]
Comment (LaneSh4d0w) [reply channels/eg.m3u:23]: Not an issue from my side, some simply do not stream on YouTube from time to time.

File: channels/jo.m3u [hunk 5]
```diff
@@ -19,7 +19,11 @@ https://jrtv-live.ercdn.net/jordanhd/jordanhd.m3u8
...
+#EXTINF:-1 tvg-id="Roya TV ARB" tvg-name="Roya TV ARB" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://i.imgur.com/Sa2nBTP.png" group-title="General",Roya TV (720p)
 https://weyyak-live.akamaized.net/weyyak_roya/index.m3u8
+#EXTINF:-1 tvg-id="Roya TV ARB" tvg-name="Roya TV ARB" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://i.imgur.com/Sa2nBTP.png" group-title="General",Roya TV (1080p)
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/c/royatv/live
 #EXTINF:-1 tvg-id="RotanaRadioJordan.jo" tvg-name="Rotana Radio Jordan" tvg-country="JO" tvg-language="Arabic" tvg-logo="http://service.mobile.rotanaradio.jo/sites/default/files/studio.png" group-title="Music",Rotana Radio Jordan (220p)
 http://188.247.86.66/RotanaRadio/OnAirStudio1/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="JO" tvg-language="Arabic" tvg-logo="" group-title="Kids",Toyor Al-Jannah
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/user/toyorlive/live
```
[the remark this answers is not in the document]
Comment (tilfizyawn) [reply channels/jo.m3u:29]: done thanks

File: channels/eg.m3u (hunk 4 above)
[the remark this answers is not in the document]
Comment (tilfizyawn) [reply channels/eg.m3u:23]: Add [Not 24/7]. Thanks

File: channels/iq.m3u [hunk 6]
```diff
@@ -61,22 +61,38 @@ https://live.imamhossaintv.com/live/ih4.m3u8
 https://ur.imamhusseintv.com/live/ih4.m3u8
 #EXTINF:-1 tvg-id="IshtarTV.iq" tvg-name="Ishtar TV" tvg-country="IQ" tvg-language="Arabic" tvg-logo="http://live.ishtartv.com/files/ishtar_animated.gif" group-title="General",Ishtar TV (1080p)
 http://176.9.102.245:1935/iShtarTV/liveStream/playlist.m3u8
-#EXTINF:-1 tvg-id="KIRKUKTV.iq" tvg-name="KIRKUK TV" tvg-country="IQ" tvg-language="" tvg-logo="" group-title="",KIRKUK TV
+#EXTINF:-1 tvg-id="KarbalaLive1.ir" tvg-name="Karbala Live 1" tvg-country="IQ" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/Karbala_tv.jpg" group-title="Religious",Karbala Live 1 (360p)
```
[the remark this answers is not in the document]
Comment (tilfizyawn) [reply channels/iq.m3u]: I moved it from ir.m3u. I've updated them from https://alkafeel.net/

File: channels/iq.m3u [hunk 7]
```diff
@@ -61,22 +61,38 @@ https://live.imamhossaintv.com/live/ih4.m3u8
...
+https://stream.alkafeel.net/live/alkafeel/rAa5PGot1/manifest.m3u8
+#EXTINF:-1 tvg-id="KarbalaLive4.ir" tvg-name="Karbala Live 4" tvg-country="IQ" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/Karbala_tv.jpg" group-title="Religious",Karbala Live 4 (360p)
```
[the remark this answers is not in the document]
Comment (tilfizyawn) [reply channels/iq.m3u]: Thanks, added  [Not 24/7] to them!

**Review by themak1985 — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Everything sounds good to me. Merging.

**Review by themak1985 — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**